modules/raft/src/test/java/org/apache/ignite/raft/TestWriteCommand.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.raft;
+
+import org.apache.ignite.network.NetworkMessage;
+import org.apache.ignite.network.annotations.Transferable;
+import org.apache.ignite.raft.client.WriteCommand;
+import org.apache.ignite.raft.messages.TestRaftMessagesFactory;
+
+/**
+ * Test WriteCommand.
+*/
+@Transferable(10)
+public interface TestWriteCommand extends NetworkMessage, WriteCommand {
+    static final TestRaftMessagesFactory MESSAGES_FACTORY = new TestRaftMessagesFactory();
```

Comment:
is it possible to move this factory and the `create` method back to the test where they are used?